### PR TITLE
React-Insta-Clone

### PR DESCRIPTION
**Pull request by McNultyPT**

### CODE REVIEW

File: instagram/src/App.js [hunk 1]
```diff
@@ -39,4 +41,13 @@ class App extends Component {
...
 }
 
+App.propTypes = {
+  username: PropTypes.string,
+  thumbnailUrl: PropTypes.string,
+  imageUrl: PropTypes.string,
+  likes: PropTypes.number,
+  timestamp: PropTypes.string,
+  comments: PropTypes.array
+}
```
Comment (clintKunz) [inline instagram/src/App.js]: App.js doesn't have any props, so it doesn't need to check propTypes. What you have in PostContainer is correct.

File: instagram/src/components/CommentSection/CommentSection.js [hunk 2]
```diff
@@ -53,4 +54,12 @@ class CommentSection extends React.Component {
     }
 }
 
+CommentSection.propTypes = {
+    likes: PropTypes.number,
+    comments: PropTypes.array,
+    username: PropTypes.string,
+    text: PropTypes.string,
+    timestamp: PropTypes.string
+}
```
Comment (clintKunz) [inline instagram/src/components/CommentSection/CommentSection.js]: CommentSection is only receiving comments, likes, and timestamp in the props object passed down from PostContainer.

Comment (clintKunz) [inline instagram/src/components/CommentSection/CommentSection.js:133]: Username and text aren't props being sent from the parent container (PostContainer), so you don't need propTypes on those two.

**Comment by clintKunz** [comment]
## Day I MVP
- [x] SearchBar, PostContainer, CommentSection
- [x] Import dummyData and iterate over it
- [x] Pass comments object to CommentSection
- [x] Check Types of data with PropTypes
- [x] Some Styling

## Day I Stretch
- [x] Add a comment button functional
- [x] Timestamp in CommentSection

## Day II MVP
- [x] Use componentDidMount() to set your data to the component's state
- [x] Add comments
- [x] Liking posts
- [x] Search

## Day II Stretch
- [ ] Persist data using localStorage
- [ ] Delete comments
- [ ] Implement ability to have “fuzzy” searches 

## Feedback 1/22
- Excellent work, keep it up. 
- See console error about 'class' in SearchBar, keep working on the functionality for the SearchBar
- See comments about propTypes

**Comment by clintKunz** [comment]
## Day III MVP
- [x] Create a <PostsPage /> component in your components/PostsContainer directory
- [x] Authentication HOC
- [x] LoginPage

## Day III Stretch
- [x] Use username when posting a comment
- [ ] Styled components
- [ ] Deploy to netlify

## Feedback 1/23
- Patrick, great job with completing the MVP for today, and styling looks great. You are doing great. And thanks for coming to the pair programming today. Try making your search a little more user friendly. After one search, how do I see all the posts again?

**Comment by McNultyPT** [comment]
Hey Clint, I'm still trying to figure out how to make the search more user friendly as well. I'll keep doing some research. If you some documentation you would recommend that would be great. Also, I just noticed that using the username to post a comment was a stretch goal, which I was able to complete. I appreciate all of your feedback!

**Comment by clintKunz** [comment]
## Day IV MVP
- [x] Change everything to be a Styled Component
- [x] Re-factor SearchBar to use styled-components
- [x] Create a Header styled-component.header that wraps your entire SearchBar
- [x] Re-factor your PostContainer component use only styled components

## Day IV Stretch
- [ ] Add the functionality to select a single Post. If a user clicks on a post, only show that post

## Feedback 1/25
- Patrick, you've worked consistent and hard this whole week. Well done on this project! I'm going to look into your search code today.

**Pull request merged**